docs/users/students.md

```diff
@@ -22,36 +24,41 @@ To add new students to a course, click the _Add New Students_ button at the top-
 3. Once you're okay with the list of students to be added, click the _Save List_ button. It'll add everyone you've listed as students in the course, together.
 
 ## Editing student details
 
-To edit a student's details, click on the student's name. You'll see all of the fields you entered in the creation form, and a few others:
-
-**Team Name**: If the student is in a team (details below), then you'll be able to edit the team's name here.
+To edit a student's details, click on the student's name. You'll see all of the fields you entered in the creation form, and one additional field:
 
-**Personal Coaches**: This list allows you to [directly assign coaches](/users/coaches#student-team-coaches) to a student (or team), which will allow those coaches to review submission from that student (or team).
-
-**Access Ends On**: If set, the course will be marked as _ended_ for the student (or team) on this date. Students will retain access to the course, and their own submissions, but they will not be able to complete new targets.
+**Personal Coaches**: This list allows you to [directly assign coaches](/users/coaches#assigning-coaches-to-students) to a student, which will allow coaches to filter submissions to review by the students they've been assigned to.
 
 ## Student actions
 
 In addition to editing a student's details, you can switch to the _Actions_ tab which lists the actions that you can take on a student:
 
+**Course Certificates**<br/>
+This will list all the certificates that the student has earned in the course. You can view each certificate by clicking on the certificate ID. You can issue certificate from here and also revoke a certificate if needed.
+
 **Has this student dropped out?**<br/>
 If you click the _Dropout Student_ button, the student will lose _all_ access to the course. Unlike students whose access to a course ended on a certain date, students who are marked as dropped out will not be able to access course content, or their own work within a course. The course will still be displayed on their dashboard page (marked as _Dropped Out_), but they will not be able to access its curriculum.
 
 ## Inactive students
 
-Students whose access to a course has ended, or who were marked as dropped out will be hidden from the main list of (active) students. To see these inactive students, click the _Inactive Students_ button at the top-right of the students page. Inactive students can be reactivated by selecting one or more students or teams, and then choosing the _Reactivate Students_ option.
+Students whose access to a course has ended by being part of ended cohort or who were marked as dropped out will be hidden from the main list of (active) students. To see these inactive students, select the _ended_ and _dropped_ status filter from the _Multiselect Dropdown_ filter at the top of the students page.
```

Comment:
`Multiselect Dropdown` is an implementation detail that the user may not understand; Make the docs user friends at all times;